conf/cassy.properties

```
@@ -1,8 +1,8 @@
 #scalar.cassy.server.port=
 #scalar.cassy.server.jmx_port=
-scalar.cassy.server.ssh_user=<username>
-scalar.cassy.server.ssh_private_key_path=/path/to/key
-scalar.cassy.server.slave_command_path=/path/to/bin
-scalar.cassy.server.storage_base_uri=s3://<some-bucket-name>
-scalar.cassy.server.metadata_db_url=jdbc:sqlite:/cassy/data/cassy.db
-scalar.cassy.server.srv_service_url=<srv-service-url>
+scalar.cassy.server.ssh_user=ben
+scalar.cassy.server.ssh_private_key_path=/Users/ben/.ssh/id_rsa
+scalar.cassy.server.slave_command_path=/Users/ben/git/cassyfork/build/install/cassyfork/bin
+scalar.cassy.server.storage_base_uri=https://cassydev.blob.core.windows.net/indetail-cassy-test
+scalar.cassy.server.storage_type=AZURE_STORAGE_BLOB
```

Let's revert that file to its original state except for this key that need to be set for aws
```suggestion
scalar.cassy.server.storage_type=AWS_S3
```